src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/constants.tsx

```diff
@@ -15,11 +13,3 @@ export type PlanTiers = typeof TierName.TEAM | typeof TierName.PRO
 export type OptionPeriod =
   | typeof TimePeriods.ANNUAL
   | typeof TimePeriods.MONTHLY
-
-export type NewPlanType =
```

Comment:
ideally we get rid of this while file at some point, might be able to when upgradeForm.js is converted